src/app/GlobalRedux/Features/GlobalSlice/index.ts

```diff
@@ -39,7 +39,7 @@ export const fetchGlobal = createAsyncThunk(
     const GLOBAL_URL = `https://api.coingecko.com/api/v3/global`;
     const response = await fetch(GLOBAL_URL);
     const data = await response.json();
-    console.log(data);
+    console.log("fetch requested:", data);
```

Comment:
no

Reply:
Gone. This was from creating a branch off PR 37 instead of master.